ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -34,6 +34,9 @@ chain:
     - name: HYPERSHIFT_INFRA_AVAILABILITY_POLICY
       default: "SingleReplica"
       documentation: "Availability policy for infrastructure services in guest cluster. Supported options: SingleReplica, HighlyAvailable, default SingleReplica"
+    - name: HYPERSHIFT_INSTANCE_TYPE
+      default: "m5.large"
```

Comment:
any reason why this is changing default from m5.xlarge -> m5.large?

Reply:
Sorry, I will change it back^^